samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -502,6 +502,10 @@ begin_assessment_msg_num_submission_2=time(s).
 begin_assessment_msg_highest=Answers from previous attempts will not be available within the assessment during subsequent attempts. Your highest score will be recorded.
 begin_assessment_msg_latest=Answers from previous attempts will not be available within the assessment during subsequent attempts. Your latest score will be recorded.
 begin_assessment_msg_average=Your average score will be recorded.
+begin_assessment_msg_attempt_autosubmit_warn_1=You have already submitted this assessment.  If you start another attempt it will be automatically submitted even if you do not answer all questions.  If you start a new attempt now, your score from this attempt will be
```

Comment:
This split may be convenient for some languages (English), but the best way to do for i18n may actually be to keep partial all together as just 2 strings (instead of 3). I think it would be good to get a little input from some translators. Spanish looks like this split *might* be okay but machine generated Japanese results a sentence that doesn't split well. @mpellicer ?

Reply:
Yeah, I agree, better if translators have the full string and full context, because they translate string by string using transifex.

Reply:
Thanks thought so. This looks good to me after this is changed to complete statements in 2 strings.

Reply:
Thanks for the suggestions.  It's cleaner this way, too.